app/assets/javascripts/select-all.js

```diff
@@ -0,0 +1,23 @@
+(function (Modules) {
+  "use strict";
+
+  Modules.SelectAll = function () {
+    var that = this;
+
+    that.start = function (element) {
```

Comment:
I would say that we should use `this` at this point, and use `that` inside the event handler.
But, as we are not using the reference to `this` in lines 13-15 we could get rid of `that`